content/docs/0.8.1/advanced-resources/deploy/airgap.md

```diff
@@ -5,6 +5,34 @@ weight: 2
 
 CSI driver components images names and tags can be found [here.](../../../architecture/#kubernetes-csi-driver-images)
 
+## Note:
+
+Using long registry URL may cause Longhorn installation error
+
+e.g Longhorn manager log 
```

Comment:
`Longhorn manager would report errors in the log when this happened:`

Reply:
fixed